### DISCUSSION TITLE
SPC support

### DISCUSSION THREAD

**Discussion by voirtimid** [Yubico/java-webauthn-server#284] [Ideas]
Hi guys, 

I was playing around with FIDO authentication and wanted to try out Secure Payment Confirmation SPC (https://www.w3.org/TR/secure-payment-confirmation/) also. I'm using this library for the server part. Do you have any plans to extend the library with SPC specific features.

Thank you in advance.

Regards,

Oliver

**Comment by emlun** [comment]
Thanks for asking! Moved to #285.
